TestProject1/TestProject1/CalculatorPageTests.cs

```diff
@@ -0,0 +1,276 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestProject1
+{
+    class CalculatorPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            driver.Url = "http://localhost:64177/Login";
+            driver.FindElement(By.Id("login")).SendKeys("test");
+            driver.FindElement(By.Id("password")).SendKeys("newyork1");
+            driver.FindElement(By.XPath("//*[@id='loginBtn']")).Click();
+        }
+        [TearDown]
+        public void CleanUP()
+        {
+            driver.Quit();
+        }
+
+        [Test]
+        // Positive Test 365
+        public void CalculatorPositiveTest365()
+        {
+
+            // Act
```

Comment:
No need to add an empty line here.
```suggestion
            // Act
```